examples/chart/teleport-kube-agent/templates/statefulset.yaml

```diff
@@ -198,10 +192,6 @@ spec:
           - name: KUBERNETES_TOKEN_PATH
             value: /var/run/secrets/tokens/join-sa-token
           {{- end }}
-          {{- if .Values.updater.enabled }}
-          - name: "TELEPORT_UPDATE_CONFIG_FILE"
-            value: "/etc/updater-config/update.yaml"
-          {{- end }}
```

Comment:
This will also break updater detection and canary update.

Reply:
thanks, was removed by mistake